lvgl.h

```diff
@@ -10,14 +10,6 @@
 extern "C" {
 #endif
 
-/***************************
```

Comment:
The issue with having version in lvgl.h is that it's including plethora of other header. In the include header chain, there's the header that's trying to use the version macro.  
So any code that's checking for the version should either be outside the standard/common lvgl.h include pool or we have circular dependencies. 

Right now, I'm using LVGL_MAJOR as a way to deprecate & change the structure layout. On LVGL8, since we allow `lv_img_set_src` the `lv_img_src_t` as a "first" byte in the structure set as '0xFF' to be able to be detected by img_get_src_type. But once we drop support for guess-my-type-void* this hack can be dropped and the struct be more compact.

Reply:
I see the problem. But do we really need this version check? In v8 we use the `LVGL_VERSION_MAJOR <= 8` branch and in v9 we change it to `#else` branch's code. LVG can't have two versions at the same time all code in LVGL should be compatible only with the current version. 

Version check makes sense in external files which are independent from LVGL.